proj/vs2017/vcpkg.json

```diff
@@ -1,4 +1,5 @@
 {
+    "builtin-baseline": "007aaced1a9d3245e28a2ba9395dca88ea890db1",
```

Comment:
What _is_ this…?

Reply:
That's a horrible thing that vcpkg requires if you want to specify an out-of-date version of a dependency. It's a commit hash for the vcpkg central repo. I put it in correctly and it *still* didn't work.